### DISCUSSION TITLE
Postgres pdo driver

### DISCUSSION THREAD

**Discussion by gui-canal** [asdf-community/asdf-php#77] [Q&A] [answered]
Hi guys, I just want to know how can I install a php version with postgres driver if it really is possible.

Thanks

**Comment by smorimoto** [comment, marked as answer]
It probably works by adding the `--with-pdo-pgsql` option [here](https://github.com/asdf-community/asdf-php/blob/master/bin/install#L71-L107), but I'm not familiar with PHP and don't know.

**Comment by smorimoto** [comment]
or you can use it by setting the `--with-pdo-pgsql` option to the `PHP_CONFIGURE_OPTIONS` variable.

```
PHP_CONFIGURE_OPTIONS="--with-pdo-pgsql=..." asdf install php <VERSION>
```

**Comment by gui-canal** [comment]
Nice, I put --with-pdo-pgsql on  bin/install, and it worked pretty well for me. 

I've tried `PHP_CONFIGURE_OPTIONS` as well, although it leads to some issues with https packages, I guess should be some config conflict in some point. 

many thanks for the tip ^^

**Comment by smorimoto** [comment]
My pleasure.
